apis/apps/v1alpha1/clusterdefinition_types.go

```diff
@@ -217,10 +308,13 @@ type ClusterDefinitionStatus struct {
 	// +optional
 	Message string `json:"message,omitempty"`
 
-	// Represents the most recent generation observed for this ClusterDefinition.
+	// Topologies this ClusterDefinition supported.
 	//
 	// +optional
-	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	Topologies string `json:"topologies,omitempty"`
+
+	// +optional
+	ServiceRefs string `json:"serviceRefs,omitempty"`
```

Comment:
Adding API comments to ServiceRefs would be better.

Reply:
done